app/views/case_history/allocation/_deallocate_released_offender.html.erb

```diff
@@ -1,4 +1,4 @@
 <%= render 'case_history/case_history',
-           title: 'Prisoner released',
+           title: 'Prisoner unallocated',
            description: '',
```

Comment:
Should this say 'Prisoner released' in the description? To be consistent with the transfers.